.github/workflows/sync-tokens-to-figma.yml

```diff
@@ -0,0 +1,25 @@
+name: Sync tokens to Figma
+on:
```

Comment:
Is this safe to do as an action? Will we be able to see the console output if something went wrong?

Reply:
Updating the variables in Figma could break; if it does, we must revert changes using the history view in Figma.  If something unexpected occurs in the script, this should be logged in the execution of the action.